src/app/modules/myinfo/myinfo.errors.ts

```diff
@@ -0,0 +1,86 @@
+import { ApplicationError } from '../core/core.errors'
+
+/**
+ * Circuit breaker is open
+ */
+export class MyInfoCircuitBreakerError extends ApplicationError {
+  constructor(message = 'Circuit breaker tripped') {
+    super(message)
+  }
+}
+
+/**
+ * Error while attempting to retrieve MyInfo data from the MyInfo API
+ */
+export class MyInfoFetchError extends ApplicationError {
+  constructor(message = 'Error while requesting MyInfo data') {
+    super(message)
+  }
+}
+
+/**
+ * Error while attempting to hash data or compare hashed data
```

Comment:
```suggestion
 * Error while attempting to hash data
```
Seems like hash comparison errors should be using `MyInfoHashDidNotMatchError`?

Reply:
nope, `MyInfoHashingError` is for when `bcrypt.compare` throws an error, whereas `MyInfoHashDidNotMatchError` is for when `bcrypt.compare` returns false.